cime_config/buildnml

```diff
@@ -152,6 +152,16 @@ def _create_control_files(case, caseroot, srcroot, confdir, inst_string, infile,
        varname_segId = "seg_id"
        varname_downSegId = "Tosegment"
        varname_pfafCode = "pfaf"
+    elif (  config['rof_grid'] == "MIZU_USRDAT" ):
```

Comment:
`ancil_dir` also needs to be set.  `ancil_dir` is directory where `fname_ntopOld` is located.   

I believe default `ancil_dir` is this. 
`<ancil_dir>                  /glade/p/cesmdata/cseg/inputdata/rof/mizuRoute/ancillary_data/     !  directory containing ancillary data (river network data)`

so add something like this?
`<ancil_dir>        "please_set_ancillary_directory_in_user_ctl_mizuroute"`

Reply:
OK, got it added in.